juno/assetlist.json

```diff
@@ -1285,6 +1285,28 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/mille.png"
       }
+    },
+    {
+      "description": "Void",
+      "type_asset": "cw20",
+      "address": "juno1lpvx3mv2a6ddzfjc7zzz2v2cm5gqgqf0hx67hc5p5qwn7hz4cdjsnznhu8",
+      "denom_units": [
+        {
+          "denom": "cw20:juno1lpvx3mv2a6ddzfjc7zzz2v2cm5gqgqf0hx67hc5p5qwn7hz4cdjsnznhu8",
+          "exponent": 0
+        },
+        {
+          "denom": "void",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:juno1lpvx3mv2a6ddzfjc7zzz2v2cm5gqgqf0hx67hc5p5qwn7hz4cdjsnznhu8",
+      "name": "Void",
+      "display": "Void",
```

Comment:
display should use lower case 'v' (to match the denom_unit)

Reply:
Fixed @JeremyParish69